python/spear/env.py

```diff
@@ -553,3 +553,96 @@ def _serialize_array(array, space, byte_order):
     assert (array <= space.high).all()
     data = array.data if byte_order is None else array.newbyteorder(byte_order).data
     return data
+
+
+# Custom Env implementation for OpenBot
+class OpenBotEnv(Env):
+
+    def __init__(self, config):
+    
+        super().__init__(config)
```

Comment:
Explicitly add `wheel_encoder` here because your code requires it. Likewise with `add_force`.

Reply:
If I understand correctly, do you want me to add `wheel_encoder` and `add_force` as part of `action_components` by default as it is required by the code?

Reply:
I'm an idiot. `add_force` is not part of the `VehicleAgent's` action space. I meant to say `set_drive_torque`.

And on second thought, rather than attempting to correct configuration errors, I think it would be simpler to just assert that the following conditions are satisfied:

```
assert config.SIMULATION_CONTROLLER.AGENT == "VehicleAgent"
assert "set_drive_torque" in config.SIMULATION_CONTROLLER.VEHICLE_AGENT.ACTION_COMPONENTS
assert "wheel_encoder" in config.SIMULATION_CONTROLLER.VEHICLE_AGENT.OBSERVATION_COMPONENTS
```